contrib/!lang/rust/packages.el

```diff
@@ -58,3 +59,12 @@
       :if (configuration-layer/package-usedp 'company)
       :defer t
       :init (push 'company-racer company-backends-rust-mode))))
+
+(defun rust/init-racer ()
+  (use-package racer
+    :if rust-enable-racer
+    :defer t
+    :init (add-hook 'rust-mode-hook
+                    '(lambda ()
+                       (racer-mode)
```

Comment:
You could probably do this by adding `:mode ("\\.rs'" racer-mode)` to the use-package macro so you only need the hook for eldoc.